lib/ddtrace/contrib/active_record/patcher.rb

```diff
@@ -40,6 +44,25 @@ def patch_active_record
           ::ActiveSupport::Notifications.subscribe('sql.active_record') do |*args|
             sql(*args)
           end
+
+          # subscribe when the active record instantiates objects
+          ::ActiveSupport::Notifications.subscribe('instantiation.active_record') do |*args|
+            instantiation(*args)
+          end
+        end
+
+        def instantiation_tracing_supported?
```

Comment:
do we want to use that also to avoid `ActiveSupport::Notifications` calls?

Reply:
Yes, will use this to prevent unnecessary instrument calls.